packages/styled-components/src/models/StyleSheetManager.js

```diff
@@ -1,13 +1,15 @@
 // @flow
-
 import React, { useContext, useMemo, type Node, type Context } from 'react';
 import PropTypes from 'prop-types';
 import StyleSheet from '../sheet';
 import StyledError from '../utils/error';
+import createStylisInstance from '../utils/stylis';
 
 type Props = {
   children?: Node,
   sheet?: StyleSheet,
+  stylisMiddlewares?: Array<Function>,
```

Comment:
Small nit but unfortunately the plural is just "middleware"; shall we call this "stylisPlugins" instead?